stanford_landing_page.views_default.inc

```diff
@@ -42,7 +42,6 @@ function stanford_landing_page_views_default_views() {
     'field_s_image_info' => 'field_s_image_info',
     'title' => 'title',
     'field_s_landing_page_layout' => 'field_s_landing_page_layout',
-    'timestamp' => 'title',
```

Comment:
You have also removed the timestamp field from being rendered with the title. Is this intentional?